docs/kura-wires/assets-as-wire-components.md

```diff
@@ -103,4 +105,31 @@ The following operations will happen:
 * The **LED2**: `78` property will have no effect, since the Asset configuration contains a channel named **LED2** with **type** = `READ_WRITE`, but **value.type** = `BOOLEAN` != `LONG`.
 * The **Toggle-4**: `true` property will have no effect, since the asset contains a channel named `Toggle-4`, with **value.type** = `BOOLEAN` but **type**  = `READ` != `WRITE` | `READ_WRITE`
 * The  **foo**: `bar` property will have no effect, since none of the defined channels has `foo` as **name**.
-* The Asset will read and emit all of the channel values with **type** = `READ` or `READ_WRITE`, since a WireEnvelope has been received.
\ No newline at end of file
+* The Asset will read and emit all of the channel values with **type** = `READ` or `READ_WRITE`, since a WireEnvelope has been received.
+
+## Wire Asset global configuration parameters
+
+The WireAsset component provides the following global (non per-channel) configuration parameters that can be used to customize component behavior:
+
+* **emit.all.channels**: Specifies whether the values of all READ or READ_WRITE channels should be emitted when a channel event is received in listen mode. If set to true, the values for all channels will be read and emitted, if set to false, only the value for the channel related to the event will be emitted.
+
+* **timestamp.mode**: Allows to configure how timestamps are emitted, the following modes are supported:
+
+  * **NO_TIMESTAMPS**: no timestamp-related properties will be emitted.
+  * **PER_CHANNEL**: the component will emit a driver-generated timestamp property per channel, as the value of the `<channel name>_timestamp` property.
+  * **SINGLE_ASSET_GENERATED**: the component will emit a single timestamp per request, generated by the Asset itself before emitting the envelope as the value of the `assetTimestamp` property.
+  * **SINGLE_DRIVER_GENERATED_MAX** and **SINGLE_DRIVER_GENERATED_MIN**: the component will emit a single driver generated timestamp being respectively the max (most recent) or min (oldest) among the timestamps of the channels as the value of the `assetTimestamp` property.
```

Comment:
The sub-list is not correctly indented.
<img width="538" alt="Screenshot 2024-10-30 at 10 37 44" src="https://github.com/user-attachments/assets/030f5ad9-0433-49ca-8d0c-7d3da4bddf03">